packages/front/src/lib/dsfr/Pagination.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import type Store from "$lib/core/Store";
+
+    export let totalPage: Store<number>;
+    export let currentPage: Store<number>;
+
+    const changePage = (page: number) => {
+        if (page < 1 || page > totalPage.value) return;
+        currentPage.set(page);
+    };
+
+    const visibleLinks: { current: boolean; pageNumber: number }[] = [];
```

Comment:
```suggestion
    const visibleLinks: { isCurrent: boolean; pageNumber: number }[] = [];
```

sinon je m'attend à trouver un nombre quand je le croise

Reply:
Done